### PR TITLE
Update SHAKE op code details

### PR DESCRIPTION
**Pull request by dangarfield**
Small change to update SHAKE op code details in line with what I updated in the wiki:
http://wiki.ffrtt.ru/index.php?title=FF7/Field/Script/Opcodes/5E_SHAKE

### CODE REVIEW

**Comment by myst6re** [comment]
Great PR, thanks!

Please indent lines with tabs.

**Comment by dangarfield** [comment]
Sorry, not sure what your message meant.
That last commit was the replacing of whitespace with tabs in the files that I originally committed.
I also updated the english description of the RETTO to make it a little clearer.

**Comment by myst6re** [comment]
See the context here: https://github.com/myst6re/makoureactor/commit/5a2987363143861de88bfeb596917c0b0c51f947

**Comment by dangarfield** [comment]
Update the 'this' to 'current'

**Comment by myst6re** [comment]
> Appellant means 'calling', but in this context I would say that the 'calling' group would be the group that envoked the currently executing script. But this is really executing the current groups' script rather than the 'caller'.
> 
> Eg, Cloud Script 2 calls Barret Script 3 using REQ. Inside Barret's script 3, a RETTO is called with op.f of 4. Barret's script 3 (and all other running scripts are returned and Barret Script 4 is run. In this instance, I would say that Cloud is the 'appellant' or calling group and Barret is the 'current scripts' group'.

I understand that, I just point a thing you fixed after, the "from the this entity" which were a strange sentence. You can write "from this entity" or "from the current entity", I don't mind :)

File: widgets/ScriptEditorWidgets/ScriptEditorGenericList.h [hunk 1]
```diff
@@ -31,7 +31,7 @@ class ScriptEditorGenericList : public ScriptEditorView
 		/*10*/layer_id, parametre_id, state_id, window_id, text_id, /*15*/item_id, materia_id, animation_id, music_id, sound_id, /*20*/movie_id,
 		operateur, keys, color, coord_x, coord_y, coord_z, window_w, window_h, window_num, window_type, window_var, direction, vitesse, vitesse2,
 		priorite, menu, jump, jump_l, rotation, quantity,
-		bank, adress, byte, word, sword, bit, boolean, label, akao, dword
```
Comment (myst6re) [inline widgets/ScriptEditorWidgets/ScriptEditorGenericList.h:34]: tabs please :)

Comment (dangarfield) [reply widgets/ScriptEditorWidgets/ScriptEditorGenericList.h:34]: Apologies, I missed that one.

**Review by myst6re — APPROVED** [review]

**Comment by myst6re** [comment]
Is it ready to merge?

**Comment by dangarfield** [comment]
Yes, I believe so, just your CI build checks, but there's nothing of real consequence in there

**Comment by myst6re** [comment]
Thanks!

**Pull request merged**